lib/ClimaLandSimulations/src/Fluxnet/fluxnet_sites/US-Var/US-Var_parameters.jl

```diff
@@ -0,0 +1,122 @@
+export vairaranch_default_params
+
+"""
+Site-specific model parameters for running CliMA LSM on the Ozark
+fluxtower site.
+"""
+vairaranch_default_params = (
+    hetero_resp = (
+        # Heterotrophic respiration parameters
+        θ_a100 = FT(0.1816),
+        D_ref = FT(1.39e-5),
+        b = FT(4.547),
+        D_liq = FT(3.17),
+        # DAMM,
+        α_sx = FT(194e3),
+        Ea_sx = FT(61e3),
+        kM_sx = FT(5e-3),
+        kM_o2 = FT(0.004),
+        O2_a = FT(0.209),
+        D_oa = FT(1.67),
+        p_sx = FT(0.024),
+    ),
+    auto_resp = (
+        # Autotrophic respiration parameters
+        ne = FT(8 * 1e-4),
+        ηsl = FT(0.01),
+        σl = FT(0.05),
+        μr = FT(1.0),
+        μs = FT(0.1),
+        f1 = FT(0.012),
+        f2 = FT(0.25),
+    ),
+    soil = (
+        # Soil parameters
+        soil_ν = FT(0.45), # m3/m3
+        soil_K_sat = FT(0.45 / 3600 / 100), # m/s,
+        soil_S_s = FT(1e-3), # 1/m, guess
+        soil_vg_n = FT(2.0), # unitless
+        soil_vg_α = FT(2.0), # inverse meters
+        θ_r = FT(0.067), # m3/m3, 
+    ),
+    soil_heat = (
+        # Soil heat transfer parameters; not needed for hydrology only test
+        ν_ss_quartz = FT(0.38),
+        ν_ss_om = FT(0.0),
+        ν_ss_gravel = FT(0.1),
+        κ_quartz = FT(7.7), # W/m/K
+        κ_minerals = FT(2.5), # W/m/K
+        κ_om = FT(0.25), # W/m/K
+        κ_liq = FT(0.57), # W/m/K
+        κ_ice = FT(2.29), # W/m/K
+        κ_air = FT(0.025), #W/m/K
+        ρp = FT(2700), # kg/m^3
+        #κ_solid = Soil.κ_solid(ν_ss_om, ν_ss_quartz, κ_om, κ_quartz, κ_minerals)
+        #κ_dry_val = Soil.κ_dry(ρp, soil_ν, κ_solid, κ_air)
+        #κ_sat_frozen_val = Soil.κ_sat_frozen(κ_solid, soil_ν, κ_ice)
+        #κ_sat_unfrozen_val = Soil.κ_sat_unfrozen(κ_solid, soil_ν, κ_liq);
+        #ρc_ds = FT((1 - soil_ν) * 4e6); # J/m^3/K
```

Comment:
why are some params commented out in this file? (and in the other parameter files)

Reply:
Only Ozark parameters file was done, I'll do the other ones now (and delete those commented out lines)